src/main/java/microsoft/exchange/webservices/data/autodiscover/AutodiscoverDnsClient.java

```diff
@@ -91,12 +91,14 @@ protected String findAutodiscoverHostFromSrv(String domain)
           "No appropriate SRV record was found.");
       return null;
     } else {
+      String hostName = dnsSrvRecord.getNameTarget();
+      if (hostName.length() > 0 && hostName.charAt(hostName.length()-1)=='.') {
```

Comment:
please use `StringUtils.endsWith(.., ..)` its doing the same but looking way clearer.